macro/derive/src/dump_decoded.rs

```diff
@@ -0,0 +1,170 @@
+//! Module with `DumpDecoded` macro implementation
+
+use super::*;
+
+#[cfg(feature = "dump_decoded")]
+static mut TYPES: Option<Vec<String>> = None;
```

Comment:
why isn't it just an empty vector as a default value?

Reply:
You are right, it can be just empty vector. I didn't notice that `Vec::new()` is *const*